Makefile.inc

```diff
@@ -11,10 +11,12 @@
 SEARCH  += $(USART)/code/headers
 SEARCH  += $(USART)/code/src
 
-
 # source files in this library
 SOURCES += $(wildcard $(USART)/code/src/*.cpp)
 
 # header files in this library
 HEADERS += $(wildcard $(USART)/code/headers/*.hpp)
 
+# search for queue.hpp
+SEARCH  += $(USART)/../datastructures/code/headers
```

Comment:
probably not necesary since the datastructures will be included via internal communication but it is save to have anyway